config/captcha.php

```diff
@@ -7,17 +7,17 @@
  * Don't edit this file directly, add the section to your config.php
  */
 return [
-    'enabled' => false,
-    'sitekey' => '',
-    'secret'  => '',
+    'enabled' => env('CAPTCHA_ENABLED', 'false'),
```

Comment:
Crap, merged this too fast. The `'false'` shouldn't be in quotes.

Reply:
it will always be enabled then ?

Reply:
No, it should still evaluate to false

Reply:
it was kind of a harmless mistake then... Fixed ;)